### DISCUSSION TITLE
`.describe()` API on tables/expressions?

### DISCUSSION THREAD

**Discussion by lostmygithubaccount** [ibis-project/ibis#7070] [Ideas]
- https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.describe.html
- https://pola-rs.github.io/polars/py-polars/html/reference/dataframe/api/polars.DataFrame.describe.html
- https://docs.dask.org/en/latest/generated/dask.dataframe.DataFrame.describe.html
- https://spark.apache.org/docs/3.1.1/api/python/reference/api/pyspark.sql.DataFrame.describe.html
- https://docs.snowflake.com/developer-guide/snowpark/reference/python/latest/api/snowflake.snowpark.DataFrame.describe
- PyStarburst dataframes have it too

while it's not practically the most useful thing, it is good to get a general overview of a table and great for demos, running more computationally expensive operations on medium-large-ish data 

(in translating the PyStarburst example they use this in the example, will exclude for now. I'm not sure if this is worth the effort needed to duplicate in all backends? but could be nice in those that support it? opening as a discussion for discussion prior to prioritization)

**Comment by cpcloud** [comment]
We have a documented [`.info()`](https://ibis-project.org/reference/expression-tables#ibis.expr.types.relations.Table.info) that serves this purpose. Does that work for your use cases?

**Reply by lostmygithubaccount** [reply]
wow fancy docs :)

I think this is good for now -- I might go through and look at what all of the other's return, I'm sure they're similar but not consistent. we might want to add some numeric calculations (mean, percentiles, etc.) to match or consider adding a proper `.describe()` method later if users request it, but I think no action here for now is good

**Reply by cpcloud** [reply]
Should we close the discussion to indicate resolution?

**Reply by lostmygithubaccount** [reply]
yep sounds good
